docs: task.previousTrace directive Documentation (#6538)
nextflow-io/nextflow: docs/process.md

}
```

In the above example, the {ref}`process-memory` is set according to previous trace record metrics. In the first attempt, when no trace metrics are available, it is set to 1 GB. In each subsequent attempt, the requested memory is doubled. 

:::{note}

> You can access many other fields relative to fields present in {ref}`trace record <trace-report>`  from the previous task attempt. See the {ref}`task.previousTrace <process-reference-task-properties>` documentation for a complete list of available fields.

Reviewer: ```suggestion
:::{note}
You can access many other fields relative to fields present in {ref}`trace record <trace-report>`  from the previous task attempt. See {ref}`task.previousTrace <process-reference-task-properties>` for a complete list of available fields.
:::
```

}
```

In the above example, the {ref}`process-memory` is set according to previous trace record metrics. In the first attempt, when no trace metrics are available, it is set to 1 GB. In each subsequent attempt, the requested memory is doubled. 

:::{note}

> You can access many other fields relative to fields present in {ref}`trace record <trace-report>`  from the previous task attempt. See {ref}`task.previousTrace <process-reference-task-properties>` for a complete list of available fields.

Reviewer: ```suggestion
Many fields from the previous task attempts are accessible. See {ref}`trace-report` for a list of available fields.
```
